### PR TITLE
MadStory generator project added

### LINKED ISSUE #113: Mad story generator JS
The Mad Libs project is a fun and interactive program that allows users to create humorous and often nonsensical stories by filling in the blanks with their own words or phrases it is a great way to practice user input, string manipulation, and basic programming concepts while providing an entertaining and creative experience for the users

The project typically involves the following components:
Story Template: Start by creating a story template that includes a narrative with specific placeholders for different parts of speech (e.g., nouns, verbs, adjectives, adverbs).

Requirements:
-Prompt the user to provide words or phrases for each placeholder. Collect and store these user inputs.
-Replace the placeholders in the story template with the user-provided words or phrases, creating a customized and often comical 
  story.
-Present the final generated story to the user, complete with their input, and let them enjoy the humorous results.

Tech stack : 
JS ,html , CSS

Screen shot:
image
image

### PR DESCRIPTION
**Pull request by hritikbhattacharya**
# Title and Issue number 

Title : MadStory generator project added

Issue No. : #113 

Code Stack :  html , javaScript , CSS

Close #113 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/MadStory_generator/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mad Libs</title>
```
Comment (sadanandpai) [inline native/machine-coding/MadStory_generator/index.html:7]: please add the tags to be added (refer other challenges)

File: native/machine-coding/MadStory_generator/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+body {
```
Comment (sadanandpai) [inline native/machine-coding/MadStory_generator/style.css:1]: import the core styles and also don't modify font family

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/MadStory_generator/style.css (hunk 2 above)
Comment (hritikbhattacharya) [reply native/machine-coding/MadStory_generator/style.css]: done.

File: native/machine-coding/MadStory_generator/index.html (hunk 1 above)
Comment (hritikbhattacharya) [reply native/machine-coding/MadStory_generator/index.html:10]: done

**Comment by sadanandpai** [comment]
@hritikbhattacharya 
Clicking on share buttons of Facebook and Linkedin show Error page

**Comment by hritikbhattacharya** [comment]
ok let me check.

**Pull request closed without merging**